client/sources/common/interpreter.py

```diff
@@ -208,13 +226,16 @@ def load(self, code, setup='', teardown=''):
         self._code = code
         self._teardown = textwrap.dedent(teardown).splitlines()
 
+        self.cases_passed = 0
+        self.cases_total = 0
+
     def interpret(self):
         """Interprets the console on the loaded code.
 
         RETURNS:
         bool; True if the code passes, False otherwise.
         """
-        if not self._interpret_lines(self._setup):
+        if not self._interpret_lines(self._setup, should_print=False):
```

Comment:
why `should_print=False`?

Reply:
to avoid printing the 4 lines of code needed for importing (makes doctests harder to read) -- since we don't want to affect the rest of okpy, I'll modify this a bit